.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
Please remove this file from the commit. Add values to .gitignore to avoid commit files this